.doc_gen/metadata/route53_metadata.yaml

```diff
@@ -0,0 +1,35 @@
+# zexi 0.4.3
+route-53_GetHostedZoneCount:
+  title: Get the number of hosted zones that are associated with the current &AWS; account using an &AWS; SDK
+  title_abbrev: Get the number of hosted zones
+  synopsis: get the number of hosted zones.
+  category:
+  languages:
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: rust_dev_preview/route53
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - route53.rust.route53-helloworld
+  services:
+    route-53: {GetHostedZoneCount}
+route-53_ListHostedZones:
+  title: Get a list of the public and private hosted zones associated with the current &AWS; account using an &AWS; SDK
```

Comment:
Shorten title and include service entity:
```
Get a list of the public and private &R53; hosted zones using an &AWS; SDK
```

Reply:
Done